tools/releasing/create_release_branch.sh

```diff
@@ -59,6 +59,22 @@ git checkout -b $target_branch
 #change version in all pom files
 find . -name 'pom.xml' -type f -exec perl -pi -e 's#<version>(.*)'$OLD_VERSION'(.*)</version>#<version>${1}'$NEW_VERSION'${2}</version>#' {} \;
 
+ultra=$(echo "${NEW_VERSION}" | perl -p -e 's#(\d+)\.\d+\.\d+#$1#')
+major=$(echo "${NEW_VERSION}" | perl -p -e 's#\d+\.(\d+)\.\d+#$1#')
+minor=$(echo "${NEW_VERSION}" | perl -p -e 's#\d+\.\d+\.(\d+)#$1#')
```

Comment:
no;  we use ?.major.minor unfortunately.

Reply:
I know that we are not using the right terminologies in Flink, but maybe this is our chance of slowly introducing new terms.

Reply:
Doing that will just introduce confusion; recent discussions on the ML showed that several devs were not aware of our version scheme, so it could easily happen that another one sees this and thinks that's what we are using.

Reply:
+1